### ISSUE TITLE
"Check Coupon" button not translatable

### ISSUE THREAD

**Issue by gio00** [enupal/stripe#96]
### Description
Coupon button "Check Coupon" is not translatable. I have copied the code responsible.

```twig
{# /_frontend/paymentFormSca.twig and  /_frontend/paymentForm.twig #}
...
<input id="check-coupon-button-{{ paymentForm.id }}" type="submit" value="Check Coupon"/>
...

```

I think it should be like this: 
```twig
...
<input id="check-coupon-button-{{ paymentForm.id }}" type="submit" value="{{ 'Check Coupon'|t }} "/>
...

```

### Additional info

- Craft version: 3.4.20
- PHP version: 7.3
- Database driver & version:
- Plugin version: 2.6.1

**Comment by andrelopez** [comment]
We'll fix this very soon. Thanks!

**Comment by andrelopez** [comment]
Hi @gio00 we just released Stripe Payments v2.6.2 with a fix to this issue and also add `checkCouponLabel` if you want to pass that via options

https://docs.enupal.com/stripe-payments/coupons/overview.html
